src/passes/Inlining.cpp

```diff
@@ -1344,7 +1344,7 @@ struct Inlining : public Pass {
     // fill in info, as we operate on it in parallel (each function to its own
     // entry)
     for (auto& func : module->functions) {
-      infos[func->name];
+      infos.try_emplace(func->name);
```

Comment:
These will definitely insert (the map is empty before), so perhaps `emplace` would be faster than `try_emplace`..? I admit I don't know if this would matter though.

The same might happen in a few other places, but this one is especially clear-cut (we clear the map right before the loop).

Reply:
I see, sounds good.

lgtm